text/0024-schema-syntax.md

```diff
@@ -183,7 +191,7 @@ action ViewDocument in [ReadActions]
   attributes {
     version: 1,
     group: "meta"
-  };
+  }
```

Comment:
why this? don't we still need the closing `;`?

Reply:
Also, I think the braces are mismatched -- `attributes` should be at the same nest level as `principal` etc.

Reply:
No, I don't think it can be, otherwise how to specify an "empty" body.

Reply:
```
action View {
  attributes: {
    readOnly: true
  }
};
```
?  

ie it just omits `principal`, `resource`, and `context`

Reply:
the alternative `action View {} { attributes: ... };` is also fine with me though

Reply:
er I mean `action View {} attributes { ... };`